third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -1048,6 +1073,62 @@ func testAccBigQueryTableFromSheet(context map[string]interface{}) string {
 `, context)
 }
 
+func testAccBigQueryTableFromBigtable(context map[string]interface{}) string {
+	return Nprintf(`
+	resource "google_bigtable_instance" "instance" {
+		name = "tf_test_bigtable_instance_%{random_suffix}"
+
+		cluster {
+			cluster_id = "%{random_suffix}"
+			zone       = "us-central1-b"
+		}
+
+		instance_type = "DEVELOPMENT"
+		deletion_protection = false
+	}
+
+	resource "google_bigtable_table" "table" {
+		name          = "%{random_suffix}"
+		instance_name = google_bigtable_instance.instance.name
+
+		column_family {
+			family = cf-"%{random_suffix}-first"
```

Comment:
Looks like this is incorrect. Quotes should wrap the `cf-`.

```
Error: Missing newline after argument
on config893095785/terraform_plugin_test.tf line 19, in resource "google_bigtable_table" "table":
19:       family = cf-"j42ngv1ayb-first"
An argument definition must end with a newline.
```

Reply:
sorry for that - fixed now.